.github/workflows/terraform.yml

```diff
@@ -0,0 +1,38 @@
+# This workflow installs the latest version of Terraform CLI. On pull request events, this workflow will run
+# `terraform init`, `terraform fmt`, and `terraform plan`.
+#
+# Documentation for `hashicorp/setup-terraform` is located here: https://github.com/hashicorp/setup-terraform
+
+name: 'Terraform'
+
+on:
+  push:
```

Comment:
Why does the comment on line 1 say "On pull request events" but lines 8-9 say "on push"?

Reply:
Thank you for spotting that discrepancy.